addon/components/from-elsewhere.js

```diff
@@ -12,12 +12,19 @@ export default Ember.Component.extend({
     }
   },
 
+  fastboot: Ember.computed(function() {
+    return Ember.getOwner(this).lookup('service:fastboot');
+  }),
+  isFastBoot: Ember.computed('fastboot', function() {
+    return this.get('fastboot') && this.get('fastboot.isFastBoot');
```

Comment:
You can just do `return this.get('fastboot.isFastBoot')` (if you do `this.get` with a path through an `undefined` value, you will receive `undefined` back and it will not error).